packages/ui/src/ToggleFormik.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactElement } from 'react'
+import { useField } from 'formik'
+import { FieldValidatorGeneric, getFieldError } from './utils/formik'
+
+import { Toggle, ToggleProps } from './Toggle'
+
+export type ToggleFormikProps = ToggleProps & {
+  name: string
+  validate?: FieldValidatorGeneric<boolean>
+}
```

Comment:
Depends on https://github.com/hazelcast/frontend-shared/pull/126/files?short_path=b082a23#r543642690
```suggestion
export type ToggleFormikProps<V extends object> = ToggleExtraProps & {
  name: ExtractKeysOfValueType<V, boolean>
  validate?: FieldValidatorGeneric<boolean>
}
```
`ExtractKeysOfValueType` allows us to check that value of the prop passed as `name` is actually `boolean` in `formik`

Reply:
done. (the type magic here is way over my head for now.)